Code/Tools/Player/CMakeLists.txt

```diff
@@ -34,6 +34,11 @@ if (EZ_CMAKE_PLATFORM_WINDOWS_UWP)
     target_link_libraries(${PROJECT_NAME} PUBLIC FmodPlugin)
   endif()
 
+  if (EZ_BUILD_OPENXR)
```

Comment:
Isn't this already covered by the finalize.cmake file in the plugin?

Reply:
The problem is that to be able to call target_link_libraries against these plugins I need to be able to resolve their generator expressions which requires calling find_package again which does not work in the finalize function as it is not cause the project to be able to resolve the generator expression probably because it's called in a different scope. I haven't found a way around this so far.

Reply:
Yeah, CMake is another technology that fails at solving the problems it is meant to solve. Let's just ignore this for the time being.